docs/guide/faq/README.md

```diff
@@ -75,6 +75,45 @@ Note that when switching from `zstack` -> `ember` or `ember` -> `zstack` re-pair
     1. Start re-pairing devices 1 by 1
 - If re-pairing was **not** required and your devices do not respond; restart some routers by removing them from the mains power for a few seconds.
 
+## How do I migrate from one adapter to another, including previously unsupported adapters (advanced users)?
```

Comment:
Maybe we can add this to and create a new page for it.

```
Migration from one adapter to another requires backup and restore support which is so far only implemented for the `zstack` (Texas Instrument) and `ember` adapters. Backup and restore is **not officially supported** for any other adapters (`conbee`, `ezsp`, `zboss` and `zigate`). However you might have success using this method (link to new page)
```

Reply:
I have moved the guide to its own page in Advanced menu and modified a few sentences.

- I added a warning to not create issue in Z2M on this method (but ok to discuss it in the discussion board)
- I added a list of confirmed "working" migration combination